R/RTconvert.R

```diff
@@ -104,7 +104,8 @@ RTconvert = function(lines, add=c(), mark.lines=FALSE, filename="", included=FAL
 #        outadd[seq(1,length(outadd),2)] = RT.linemark(n$start.line:n$end.line, filename)
       }
     } else if (tag == "RT") {
-      outadd = RTfile(gsub(' ', '', lu), c(), mark.lines, TRUE );
+      includedFile = findFileInDirs(gsub(' ', '', lu), includedirs)
+      outadd = RTfile(includedFile, c(), mark.lines, TRUE );
```

Comment:
I think it's best here to call by named arguments:
`RTfile(includedFile, mark.lines=mark.lines, included=TRUE, includedirs=includedirs )`

Reply:
Yea, good idea